### ISSUE TITLE
Anzeige Glossarbeiträge pro Page beschränken

### ISSUE THREAD

**Issue by DanEi75** [Sioweb/Glossar#34]
Toll wäre, wenn die Anzahl der Glossarbeiträge innerhalb der Pagination beschränkt werden könnte, ähnlich wie bei der Nachrichtenliste. Also, z.B. zeige 5 Beiträge pro Seite an. 

Innerhalb der A-Z-Pagination wäre das dann eine weitere Einteilung. Gerade bei umfangreichen Glossars könnte das sinnvoll sein.

**Comment by Sioweb** [comment]
Stimmt, in der Contao 4 Version habe ich das auch schon eingebaut. Die Version 3 habe ich bislang eher vernachlässigt.
